engine/src/main/java/org/terasology/engine/subsystem/DisplayDevice.java

```diff
@@ -15,11 +15,13 @@
  */
 package org.terasology.engine.subsystem;
 
+import org.terasology.module.sandbox.API;
 import org.terasology.rendering.nui.layers.mainMenu.videoSettings.DisplayModeSetting;
 import org.terasology.utilities.subscribables.Subscribable;
 
 import java.util.List;
 
+@API
```

Comment:
Howcome this goes API?

Reply:
It needs needs for rendering module now.
This class provides window width and height.